README.md

```diff
@@ -1,6 +1,34 @@
 # notation-akv
 
-Azure Provider for Notation CLI
+Azure Provider for the Notary v2 [Notation CLI](https://github.com/notaryproject/notation)
+
+## Getting Started:
+The following summarizes the steps to configure the Azure Key Vault notation plugin, configure gatekeeper, sign and verify a container image to Azure Kubernetes Service
+
+```bash
+# Configure notation with the Azure Key Vault plugin
+notation plugin add azure-kv ~/.config/notation/plugins/azure-kv/notation-azure-kv
+
+# Add signing and verification keys to the notation configuration policy
+notation key add --name $KEY_NAME --plugin azure-kv --id $KEY_ID --kms
+notation cert add --name $KEY_NAME --plugin azure-kv --id $KEY_ID --kms
+
+# Install ratify, with the verification key
+helm install ratify ratify/charts/ratify \
+        --set registryCredsSecret=regcred \
+        --set ratifyTestCert=$PUBLIC_KEY
+kubectl apply -f ./ratify/charts/ratify-gatekeeper/templates/constraint.yaml
```

Comment:
As this file is not in this repo, we should provide the absolute path to the raw content in github? or should we add those files as example in this repo?

Reply:
This readme summary isn't meant to be executed as there are several additional commands required. I struggled a bit with the quick summary, as opposed to the link to the docs below.